Tests/EndToEndTests/EndToEndTests.swift

```diff
@@ -0,0 +1,50 @@
+import class Foundation.Bundle
+import XCTest
+
+final class EndToEndTests: XCTestCase {
+  func testCSS() throws {
+    // Some of the APIs that we use below are available in macOS 10.13 and above.
+    guard #available(macOS 10.13, *) else {
+      return
+    }
+
+    let process = Process()
+    process.executableURL = productsDirectory.appendingPathComponent("swift-doc")
+    process.arguments = ["generate", "--module-name", "SwiftDoc", "--format", "html", "Sources"]
+
+    let pipe = Pipe()
+    process.standardOutput = pipe
+
+    try process.run()
+    process.waitUntilExit()
+
+    let data = pipe.fileHandleForReading.readDataToEndOfFile()
+    let output = String(data: data, encoding: .utf8)
+
+    XCTAssertEqual(output, "")
+
+    let cssPath = productsDirectory
+      .deletingLastPathComponent()
+      .deletingLastPathComponent()
+      .appendingPathComponent("documentation")
+      .appendingPathComponent("all.css")
+    let cssData = try Data(contentsOf: cssPath)
+    guard let css = String(data: cssData, encoding: .utf8) else {
+      return XCTFail("Failed to decode a UTF-8 string from `cssData` in \(#function)")
+    }
+
+    XCTAssertTrue(css.starts(with: ":root"))
```

Comment:
```suggestion
    XCTAssertTrue(css.contains(":root"))
```

Reply:
I could see this assumption breaking unexpectedly, for example if we were to add a header comment or the CSS preprocessor tool decided to reorder or remove the rules. `contains` would be a suboptimal choice if we were worried about performance, but I think this is fine here.